res/functions.sh

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env bash
+set -eux
+
```

Comment:
:+1: 

Don't spend too much effort on the scripts, I have pending PRs that changes everything in these:
https://github.com/netobserv/network-observability-cli/pull/3
https://github.com/netobserv/network-observability-cli/pull/4

We should consider merging them before fixing

Reply:
ok since this repo has bash dependency I thought having lint for bash will be add good value